fides-aws-ecs/ecs-privacy-center.tf

```
@@ -51,6 +51,17 @@ locals {
       {
         name  = "FIDES_PRIVACY_CENTER__IS_OVERLAY_ENABLED"
         value = tostring(var.privacy_center_is_overlay_enabled)
+      },
+      {
+        name  = "FIDES_PRIVACY_CENTER__USE_API_CONFIG"
+        value = tostring(var.privacy_center_use_api_config)
+      },
+var.privacy_center_root_property_path != null ? {
+  name  = "FIDES_PRIVACY_CENTER__ROOT_PROPERTY_PATH"
+  value = var.privacy_center_root_property_path
```

```suggestion
      var.privacy_center_root_property_path != null ? {
        name  = "FIDES_PRIVACY_CENTER__ROOT_PROPERTY_PATH"
        value = var.privacy_center_root_property_path,
```

You might need to run `terraform fmt -recursive` on this, but I realized that I forgot a comma to make it a proper list item.